playbooks/openshift-node/private/join.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - name: Evaluate bootstrapped nodes
   hosts: localhost
+  become: no
```

Comment:
Don't handle become for the user.

Reply:
Right, this is set here to have files from master copied locally - we don't need `become` on localhost